cmd/govim/gopls.go

```diff
@@ -137,6 +137,8 @@ func (g *govimplugin) startGopls() error {
 	initParams.Capabilities.Workspace.DidChangeConfiguration.DynamicRegistration = true
 	initParams.Capabilities.Workspace.DidChangeWatchedFiles.DynamicRegistration = true
 
+	initParams.Capabilities.Window.WorkDoneProgress = true
```

Comment:
Is it possible to tell the server about new capabilities at runtime?

Ideally we would only say we can do `WorkDoneProgress` if we actually require it.

Reply:
`gopls` doesn't support that :(. The overhead of having progress callbacks instead of logmessage callbacks should be negligible.